Packs/Slack/ReleaseNotes/3_4_20.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Slack v3
+
+- Updated the **slack-send-file** command to be compatible with all apps - including newly created apps from May 8, 2024.
```

Comment:
```suggestion
Updated the ***slack-send-file*** command to be compatible with all apps - including newly created apps from May 8, 2024.
```

Reply:
Thank you for the feedback, I have now updated the Release Notes based on your suggestion